doc/Quality_white_wine_predictor.md

```diff
@@ -0,0 +1,245 @@
+Quality white wine predictor
+================
+
+-   [Summary](#summary)
+-   [Goal](#goal)
+-   [Introduction](#introduction)
+-   [Data](#data)
+-   [Methods](#methods)
+-   [Analysis](#analysis)
+    -   [**Data Cleaning and
+        Preprocessing**](#data-cleaning-and-preprocessing)
+    -   [**EDA first conclusions**](#eda-first-conclusions)
+    -   [**Cross Validation**](#cross-validation)
+-   [Results & Discussion](#results--discussion)
+    -   [**Evaluation**](#evaluation)
+    -   [**Discussion**](#discussion)
+    -   [References](#references)
+
+# Summary
+
+This report uses the white wine database from “vinho verde” to predict
+the quality based on physicochemical properties. Quality is a subjective
+measure, given by the average grade of three experts.
+
+Before starting the predictions, the report makes a Explanatory data
```

Comment:
I recommend rephrasing to "the report performs an exploratory data analysis (EDA)".